conda/recipes/cudf/recipe.yaml

```
@@ -117,10 +117,17 @@ requirements:
         then: cudatoolkit
 
 tests:
-  - python:
-      imports:
-        - cudf
-      pip_check: false
+  python:
+    imports:
+      - cudf
+    pip_check: false
+  commands:
+    - python -c "import cudf; print(cudf.__version__)"
+    - python -m cudf.pandas -c "import pandas as pd;print(pd)"
+    - python -m cudf.pandas -c "import pandas as pd;print(pd.Series([1, 2, 3]))"
+    - RAPIDS_NO_INITIALIZE=1 python -m cudf.pandas -c "import pandas as pd;print(pd.Series([1, 2, 3]))"
+    - CUDF_NO_INITIALIZE=1 python -m cudf.pandas -c "import pandas as pd;print(pd.Series([1, 2, 3]))"
+
```

```suggestion
  - python:
      imports:
        - cudf
      pip_check: false
  - script:
      - python -c "import cudf; print(cudf.__version__)"
      - python -m cudf.pandas -c "import pandas as pd;print(pd)"
      - python -m cudf.pandas -c "import pandas as pd;print(pd.Series([1, 2, 3]))"
      - RAPIDS_NO_INITIALIZE=1 python -m cudf.pandas -c "import pandas as pd;print(pd.Series([1, 2, 3]))"
      - CUDF_NO_INITIALIZE=1 python -m cudf.pandas -c "import pandas as pd;print(pd.Series([1, 2, 3]))"
```

`rattler-build` has slightly different syntax for defining different kinds of tests: https://rattler.build/latest/reference/recipe_file/#tests-section